### DISCUSSION TITLE
Is Serverpod stable for production

### DISCUSSION THREAD

**Discussion by wailashraf71** [serverpod/serverpod#317] [Q&A] [answered]
Hi there, I'm beginning a new local network project next 2 days and I was wondering if I can depend on this package or framework instead of Node.js for simple server side operations

Can I use sockets or realtime observers to deliver data from flutter database ex: hive or Isar?

Does this framework work on desktop perfectly (Windows)?

Please respond ASAP because I really loved the hard working you did with the documentations.

Thanks in advance.

**Comment by vlidholt** [comment, marked as answer]
Hi! Serverpod is very close to a 1.0 release. The main things we are working on are documentation and tutorials.

Serverpod has built in support for Postgres (which is required to run it) and Redis (optional), but you can use any database that has Dart drivers. There is very good support for real-time communication through websockets. See this example: https://pixorama.live/

Most of the code is 100% Dart so it will run on Windows no problem. There are some non-essential tooling that currently doesn't run on Windows, so Mac is recommended for development.

**Reply by wailashraf71** [reply]
That's cool, loved it

But what about packaging and distributing? I need to make a simple installer that install the server and the flutter desktop on the machine ex: POS System that have a mobile apps connected to it through router. is that possible? Or the server is just for development and I should run docker and install the conatiners on every machine?

Thanks again.

**Reply by vlidholt** [reply]
Typically, you deploy your server to the cloud. There are Terraform scripts provided for AWS here: https://docs.serverpod.dev/deployments/deploying-to-aws

Not exactly sure what you are trying to do, do you want to run the server locally together with your desktop app? That may be more complex, but it would be complex regardless of which platform you choose.

**Reply by wailashraf71** [reply]
Well, I'm trying to create a desktop app with a backend and database (All using Flutter), and then connect other flutter apps on mobile phones to the same server on the same network through a router. Like a cashier system

Is that possible to be distributed as one executable program on desktop?

**Reply by vlidholt** [reply]
I haven’t got too much experience from Windows. It’s more of a question if you can bundle and run Postgres with your app.

**Comment by shurbajix** [comment]
is there any possibility to use serverpod  to build a big project I mean is stable for that 
And thank you for That
